src/libaktualizr/package_manager/packagemanagerconfig.h

```diff
@@ -18,6 +18,7 @@ struct PackageConfig {
 
   // Options for simulation (to be used with kNone)
   bool fake_need_reboot{false};
+  bool force_reboot{false};
```

Comment:
Grouped like this, it looks like it's only for simulation (kNone which is the fake package manager). Maybe put it before, just after packages_file and keep this order in updateFromPropertyTree and writeToStream.

Reply:
OK.